scripts/lib/helm.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+add_helm_repo() {
+   if ! should_execute add_helm_repo; then
+     return 1
+   fi
+
+   # add the helm repository containing charts for starting AWS service controllers
+   if ! helm repo add "$HELM_LOCAL_REPO_NAME" "$HELM_REPO_SOURCE" > /dev/null 2>&1; then
+     echo "Unable to add local helm repo from '$HELM_REPO_SOURCE'"
+     TEST_PASS=1
+     return 1
+   fi
+
+  #list the charts in the the local repo
+  echo "Validating the presence of '$HELM_REPO_CHART_NAME' in local repo '$HELM_LOCAL_REPO_NAME' "
+  helm_repo_output_lines=$(helm search repo "$HELM_LOCAL_REPO_NAME" | grep "$HELM_REPO_CHART_NAME" | wc -l)
+  if [[ "$helm_repo_output_lines" -gt 0 ]]; then
+    echo "'$HELM_REPO_CHART_NAME' chart is present in local helm repo '$HELM_LOCAL_REPO_NAME'"
+  else
+    echo "'$HELM_REPO_CHART_NAME' chart is NOT present in local helm repo '$HELM_LOCAL_REPO_NAME'."
+    TEST_PASS=1
+  fi
+}
+
+install_helm() {
+  # install helm in /tmp directory
+  pushd /tmp
+  # clone the source
+  git clone https://github.com/helm/helm.git
```

Comment:
This is cloning the master branch, IMO it would be better if we use a stable release instead.